### PR TITLE
WIKI-1207:Missing images in the exported file for private pages

### PR DESCRIPTION
**Pull request by azaoui**

### CODE REVIEW

**Comment by boubaker** [comment]
Conserve the original formatting and don't use tabs plz

File: wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java [hunk 1]
```diff
@@ -131,9 +134,17 @@ public String getImageURL(ResourceReference imageReference, Map<String, String>
         page = wikiService.getExsitedOrNewDraftPageById(wikiMarkupContext.getType(), wikiMarkupContext.getOwner(), wikiMarkupContext.getPageName());
 
         Attachment att = wikiService.getAttachmentOfPageByName(attachmentName, page);
-        if (att != null) {
-          sb.append(att.getDownloadURL());
-        }
+		if (att != null) {
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java]: Indentation is not good.

File: wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java [hunk 2]
```diff
@@ -131,9 +134,17 @@ public String getImageURL(ResourceReference imageReference, Map<String, String>
...
+			if (att.getMimeType().contains("image")) {// only for image
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java]: We should probably test "starts by image/" to have a more robust test.

File: wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java [hunk 3]
```diff
@@ -131,9 +134,17 @@ public String getImageURL(ResourceReference imageReference, Map<String, String>
...
+				sb.setLength(0);// clean up the sp
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java]: sp -> sb

File: wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java (hunk 2 above)
Comment (thomasdelhomenie) [reply wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java]: Please change "image" by "image/"

Comment (AtiaZ) [reply wiki-service/src/main/java/org/exoplatform/wiki/rendering/impl/DefaultWikiModel.java]: done

**Pull request closed without merging**